### PR TITLE
[5.3] KZOO-410: UUID treatment for command selector

### PR DESCRIPTION
**Pull request by kdamas93**
Is necessary implement/automate the use of UUID for the command selector for its _id.
To do this, it is necessary to receive the uuid field in the payload through the console and in the URL from the API and this will be validated and stored in the _id and uuid fields documents.

### CODE REVIEW

File: kazoo_documents/src/kazoo_ui_apps_maintenance.erl [hunk 1]
```diff
@@ -245,12 +245,12 @@ find_metadata(AppPath) ->
...
-            case kz_doc:id(JSON) of
-                'undefined' ->
+            case kz_json:is_defined(<<"uuid">>, JSON) of
+                'false' ->
                     {'ok', kz_doc:set_type(ValidatedJObj, kzd_applications:pvt_type())};
-                AppId ->
+                'true' ->
                     Setters = [{fun kz_doc:set_type/2, kzd_applications:pvt_type()}
-                              ,{fun kz_doc:set_id/2, AppId}
+                              ,{fun kz_doc:set_id/2, kz_json:get_value(<<"uuid">>, JSON)}
```
Comment (jamesaimonetti) [inline kazoo_documents/src/kazoo_ui_apps_maintenance.erl]: You could just `get_ne_binary_value(<<"uuid">>` and use `undefined` as the false case instead.

**Comment by jamesaimonetti** [comment]
Update PR title and summary to project standards

**Comment by kdamas93** [comment]
> Update PR title and summary to project standards

Sorry @jamesaimonetti , I did it through cherry pick and I didn't notice, I'll correct it right now.

File: kazoo_documents/src/kazoo_ui_apps_maintenance.erl (hunk 1 above)
Comment (kdamas93) [reply kazoo_documents/src/kazoo_ui_apps_maintenance.erl]: change added !

Comment (Navoda626) [reply kazoo_documents/src/kazoo_ui_apps_maintenance.erl]: @kdamas93 I think James suggesting to use `get_ne_binary_value(<<"uuid">>` in the above case in line 248. @jamesaimonetti please correct me if I'm wrong

**Review by icehess — APPROVED** [review]

**Pull request merged**